plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block-calculate-shipping.spec.js

```diff
@@ -136,12 +140,20 @@ test.describe(
 			async ( { page, context, cartBlockPage } ) => {
 				await context.clearCookies();
 
+				//  Do we need to clear localStorage and sessionStorage? Something is remembering the address.
+				await page.evaluate( () => {
```

Comment:
Can remove this comment. not sure what is going on, but guest cart tests still seem to have a country set somewhere.